example/package-lock.json

```diff
@@ -0,0 +1,16219 @@
+{
```

Comment:
we're using yarn for development, please remove package-lock

Reply:
also there are a lot of changes in `yarn.lock` and I don't get why, maybe because you've used npm?

Reply:
yes, I'm used to use npm, I will remove all the packages-lock.json